### PR TITLE
Add linkchecking

### PR DESCRIPTION
**Pull request by brianhlin**

### CODE REVIEW

File: travis/run_on_non_pull_requests.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
+#!/bin/bash -xe
+
+openssl aes-256-cbc -K $encrypted_8bf0fadbb437_key -iv $encrypted_8bf0fadbb437_iv -in deploy-key.enc -out deploy-key -d
```
Comment (bbockelm) [inline travis/run_on_non_pull_requests.sh]: I assume the point of factoring this out is that we'll soon move this to a git sub-module?

If so, I think the environment variable name is per-repo: might want to separate this particular line out of the script?

File: travis/run_on_non_pull_requests.sh [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+chmod 600 deploy-key
+eval `ssh-agent -s`
+ssh-add deploy-key
+git config user.name "Automatic Publish"
+git config user.email "djw8605@gmail.com"
```
Comment (bbockelm) [inline travis/run_on_non_pull_requests.sh]: Maybe `goc@opensciencegrid.org`?

File: travis/linkchecker.config [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+status=0
+warnings=0
+checkextern=1
+ignore=
+    ^mailto:
+    chtc.wisc.edu
+    jira.opensciencegrid.org
+    twiki.org
+    batlab.org
+    confluence.grid.iu.edu
```
Comment (bbockelm) [inline travis/linkchecker.config]: It's not clear why some of these are ignored.  Is it possible to add a few comments or examples?

In particular, seems like we would want to notice-'n'-nuke links to confluence!

File: docs/site-planning.md [hunk 4]
```diff
@@ -164,9 +164,7 @@ We recommend to use a dedicated server for hosting the shared file system. The e
...
-
-Condor provides mechanisms which can be used to greatly minimize NFS usage [NFSLite](http://vdt.cs.wisc.edu/releases/2.0.0/notes/Globus-CondorNFSLite-Setup.html). This package makes it unnecessary to have NFS-mounted **HOME** directories on all the worker nodes. Security is also improved because user proxies will not be sent across NFS in the clear. The running job will only have access to grid or voms proxies running at the same time on the same worker node as the job itself avoiding identity theft.
+NFS is known to be an *easy* but never the less %RED%inadequate%ENDCOLOR% choice for all but the smallest sites! If you want to use NFS you should read the chapter *"Optimizing NFS Performance"* in the <http://nfs.sourceforge.net/nfs-howto/>. Software partitions that can be locally installed, such as OSG\_GRID, should be locally installed and not shared unless you have an enterprise-class NFS server.
```
Comment (bbockelm) [inline docs/site-planning.md]: That sentence is easy to parse incorrectly!

I initially read it as "easy but *never* the less-inadequate".  I think you meant "easy but never-the-less inadequate".  Dropping the double-negatives, "easy but less adequate for large sites" would do.

**Comment by brianhlin** [comment]
How's it look now @bbockelm?

**Pull request merged**